### PR TITLE
fix: Escaping Jinja logical statements from assets

### PR DESCRIPTION
**Pull request by Vitor-Avila**
This PR implements escaping for Jinja logical statements (https://github.com/preset-io/backend-sdk/issues/188), and also adds two new flags:
* `--disable-jinja-templating`: This flag was added to the `sync native` command, so that the CLI doesn't render any Jinja templating from the assets. This is useful in case the assets contain Jinja and were exported via the UI (and therefore the syntax wasn't escaped).
* `--disable-jinja-escaping`: This flag was added to the `export-assets` command, so that the CLI doesn't escape Jinja syntax from the YAML files. This is useful in case the assets contain Jinja and would be imported through the UI.

Also, these two flags can be used in case the escaping/rendering logic is incorrectly escaping/rendering incorrect values (like JSON curly brackets, etc).

### CODE REVIEW

File: src/preset_cli/cli/superset/sync/native/command.py [hunk 1]
```diff
@@ -171,11 +188,17 @@ def native(  # pylint: disable=too-many-locals, too-many-arguments
         if path_name.is_dir() and not path_name.stem.startswith("."):
             queue.extend(path_name.glob("*"))
         elif is_yaml_config(relative_path):
-            config = render_yaml(path_name, env)
+            if disable_jinja_templating:
+                config = load_yaml(path_name)
+            else:
+                config = render_yaml(path_name, env)
```
Comment (betodealmeida) [inline src/preset_cli/cli/superset/sync/native/command.py:194]: Small nit:

```python
config = load_yaml(path_name) if disable_jinja_templating else render_yaml(path_name, env)
```

File: src/preset_cli/cli/superset/sync/native/command.py [hunk 2]
```diff
@@ -171,11 +188,17 @@ def native(  # pylint: disable=too-many-locals, too-many-arguments
...
 
             overrides_path = path_name.with_suffix(".overrides" + path_name.suffix)
             if overrides_path.exists():
-                overrides = render_yaml(overrides_path, env)
+                if disable_jinja_templating:
+                    overrides = load_yaml(overrides_path)
+                else:
+                    overrides = render_yaml(overrides_path, env)
```
Comment (betodealmeida) [inline src/preset_cli/cli/superset/sync/native/command.py:201]: Same here.

**Review by betodealmeida — APPROVED** [review]
sagan

File: src/preset_cli/cli/superset/sync/native/command.py (hunk 1 above)
Comment (Vitor-Avila) [reply src/preset_cli/cli/superset/sync/native/command.py]: thanks! changed as suggested

File: src/preset_cli/cli/superset/sync/native/command.py (hunk 2 above)
Comment (Vitor-Avila) [reply src/preset_cli/cli/superset/sync/native/command.py]: changed this one as well.

**Pull request merged**